### ISSUE TITLE
Titanium vein doesn't drop crystallized fire/water/air or has too low drop rate

### ISSUE THREAD

**Issue by Brutty1045** [GameFreedomGG/Sindragosa#198]
Decription:I'm sure it's not like on other servers i've played before

How it works:

How it should work:

Source (you should point out proofs of your report, please give us some source):https://www.wowhead.com/object=191133/titanium-vein

**Comment by MissZay** [comment]
yes drop rate is way too low, it's the same for herbs and some other things, I'm starting to believe it was done on purpose

**Comment by Godmatik** [comment]
Can close, Jimmy fixed this on 11/24
